rundeckapp/grails-app/jobs/rundeck/quartzjobs/ExecutionJob.groovy

```diff
@@ -251,6 +251,12 @@ class ExecutionJob implements InterruptableJob {
             initMap.secureOpts=jobDataMap.get("secureOpts")
             initMap.secureOptsExposed=jobDataMap.get("secureOptsExposed")
             initMap.execution = Execution.get(initMap.executionId)
+            def jobArguments=initMap.frameworkService.parseOptsFromString(initMap.execution.argString)
```

Comment:
this needs to happen after the below "retry" section where it attempts to load the execution again, otherwise it may fail

Reply:
https://github.com/rundeck/rundeck/pull/3498/files#diff-7ec10a60f137d2c77c4da486dbc02452R281